src/server.c

```diff
@@ -4501,6 +4501,25 @@ void bytesToHuman(char *s, unsigned long long n) {
     }
 }
 
+/* Characters we sanitize on INFO output to maintain expected format. */
+static char unsafe_info_chars[] = ":\n\r";
+static char escaped_info_chars[] = "___";   /* Must be same length as above */
```

Comment:
```suggestion
static char replacement_chars[] = "___";   /* Must be same length as above */
```

I think "escape" isn't the right word for this.

Reply:
@zuiderkwast Thanks, you're right - left this one out